rskj-core/src/main/java/org/ethereum/vm/PrecompiledContracts.java

```diff
@@ -91,6 +97,7 @@
             RIPEMPD160_ADDR,
             IDENTITY_ADDR,
             BIG_INT_MODEXP_ADDR,
+            BLAKE2F_ADDR,
```

Comment:
Precompiled contracts created after the network creation shouldn't be added here because it forks the network. In this case, pre existing number 1 blocks will have an incorrect parent hash.

Check this example as to how to implement it:
https://github.com/rsksmart/rskj/commit/e0e2c8652af0930b12ad7a31a0d329821674ae12

They are activated based on an RSKIP which is tied to a NetworkUpgrade which is tied to a block number.
So nodes update their code and when a certain block number is reached the new configuration loads.

As for what RSKIP to use, just define a non existing one, we'll handle the change when we merge it into master, for the NetworkUpgrade use `wasabiPlusOne`.

Reply:
Thanks for clarifying, i updated and defined `RSKIP153`